src/Platform/Microsoft.Testing.Platform/CommandLine/CommandLineHandler.cs

```diff
@@ -102,7 +102,7 @@ async Task DisplayPlatformInfoAsync()
             await outputDevice.DisplayAsync(this, new TextOutputDeviceData($"  Runtime location: {runtimeLocation}"));
 #endif
 
-            string moduleName = _testApplicationModuleInfo.GetCurrentTestApplicationFullPath();
+            string moduleName = _testApplicationModuleInfo.TryGetCurrentTestApplicationFullPath() ?? _testApplicationModuleInfo.TryGetAssemblyName() ?? "unknown";
```

Comment:
Same as the comment in `TestApplication.cs`. This is only for display, so we can tolerate null path and fallback to assembly name, or finally fallback to "unknown"

Reply:
Same comment about `unknown` + maybe let's introduce a constant so we have all places matching.

Reply:
Let's make some test (manual is fine) with and without `UseAppHost` to confirm we are not impacted by https://github.com/dotnet/runtime/issues/3704

Reply:
`AppContext.BaseDirectory` is used by `GetCurrentTestApplicationDirectory` not `TryGetCurrentTestApplicationFullPath`. Is this comment on the wrong line?

I'm not sure how we can reach the code to use AppContext.BaseDirectory, because in normal scenarios we are currently supporting, `TryGetCurrentTestApplicationFullPath` already works fine.
The code for AppContext.BaseDirectory will be reachable if we support Wasm in future. So not sure how to test.

Reply:
Ok let's ignore then.